packages/hawtio/src/plugins/camel/icons/components/AggregateIcon.tsx

```diff
@@ -0,0 +1,13 @@
+// === THIS FILE WAS AUTO-GENERATED BY create-route-icon.sh ===
```

Comment:
Looking at this again, I wonder why is it necessary to generate a separate React component file for each svg icon?  If you look at the contents of the `XxxIcon` component, they are all the same, just the `svg` and `size` parameters are different. If so, why not just create generic `CamelIcon` components and receive the `svg` and `size` parameters in `CamelIconProps`?

Note in the `camel/icons/index.ts` you can also pass a React element instead of component like this:
```
componentMap.set('AggregateIcon', <CamelIcon svg={aggregateIcon} size={16} />)
```

There are already many svg files and if we generate a React component for each svg, the number of files we'd need to maintain is doubled. Plus we'd need to maintain the scripts to generate those files.

Please see how we handle multiple image files in a different place:
https://github.com/hawtio/hawtio-next/blob/main/packages/hawtio/src/ui/page/HawtioBackground.tsx

Reply:
Refactored, as you suggested, with IconNames enum and generic CamelImageIcon component. Single generator javascript located in top-level package utils directory (added to linting).